.github/workflows/build-and-publish-release-artifacts.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and Publish Release Artifacts
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+
+  build-and-publish:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ^1.17.0
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v3.0.2
+
+    - run: mkdir roadmap-graphviz-linux-amd64 && env GOOS=linux GOARCH=amd64 go build -o roadmap-graphviz-linux-amd64 ./... && zip -r roadmap-graphviz-linux-amd64.zip roadmap-graphviz-linux-amd64
```

Comment:
Would you mind if we shipped these artifacts as `roadmap-$GOOS-$GOARCH.zip` since it includes tools other than `roadmap-graphviz` (and in general I'd prefer pushing folks to use the common `roadmap` CLI for all of those tasks).

Reply:
Fixed.  You may want to update your README to mention the `roadmap` CLI. It seems to tell users to use the `roadmap-graphviz` command. 
![image](https://user-images.githubusercontent.com/4627837/171713400-5ad0c46b-c8c7-4efd-9b43-932227cf05b2.png)